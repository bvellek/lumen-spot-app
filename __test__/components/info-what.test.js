import React from 'react';
import renderer from 'react-test-renderer';

import InfoWhat from '../../public/js/components/info-what';

describe('<InfoWhat />', () => {
  it('it renders correctly', () => {
    const component = renderer.create(
      <InfoWhat />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
