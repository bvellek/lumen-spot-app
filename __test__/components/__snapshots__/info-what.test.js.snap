// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoWhat /> it renders correctly 1`] = `
<section
  className="info-section"
>
  <div
    className="what-section"
  >
    <h2>
      What?
    </h2>
    <p>
      Light is EVERYTHING in photography. At Lumen Spot, we aim to make finding the perfect light as simple as possible. Enter a location of interest and find out the sun position for the day and get inspired by the vision of other photographers at thatÂ location!
    </p>
  </div>
</section>
`;
